lib/common/ipc_client.c

```diff
@@ -1169,6 +1169,7 @@ internal_ipc_get_reply(crm_ipc_t *client, int request_id, int ms_timeout,
     time_t timeout = 0;
     int32_t qb_timeout = -1;
     int rc = pcmk_rc_ok;
+    int reply_id = 0;
```

Comment:
`int32_t`, not that it matters in practice
not unreasonable to use the same data type as `request_id`, but also not necessary for comparison

Either way make the `printf` specifiers consistent